src/webapi/document.rs

```diff
@@ -67,4 +68,16 @@ impl Document {
             ).into_reference_unchecked()
         }
     }
+
+    /// Returns the <body> or <frameset> node of the current document, or null if no such element exists.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Document/body)
+    // https://html.spec.whatwg.org/#dom-document-body
+    pub fn body( &self ) -> HtmlElement {
```

Comment:
The specs say that the return type of this is `HTMLElement?`, so shouldn't this be `Option< HtmlElement >`?